--- /usr/share/plasma/plasmoids/org.kde.plasma.notifications/contents/ui/JobDelegate.qml
+++ /usr/share/plasma/plasmoids/org.kde.plasma.notifications/contents/ui/JobDelegate.qml
@@ -68,21 +68,6 @@ Column {
             }
             textFormat: Text.PlainText
         }
-
-        PlasmaComponents.ToolButton {
-            id: expandButton
-            iconSource: checked ? "arrow-down" : (LayoutMirroring.enabled ? "arrow-left" : "arrow-right")
-            tooltip: checked ? i18nc("A button tooltip; hides item details", "Hide Details")
-                             : i18nc("A button tooltip; expands the item to show details", "Show Details")
-            checkable: true
-            onCheckedChanged: {
-                if (checked) {
-                    // Need to force the Loader active here, otherwise the transition
-                    // doesn't fire because the height is still 0 without a loaded item
-                    detailsLoader.active = true
-                }
-            }
-        }
     }
 
     Loader {
@@ -91,14 +76,14 @@ Column {
         height: 0
         //visible: false // this breaks the opening transition but given the loaded item is released anyway...
         source: "JobDetailsItem.qml"
-        active: false
+        active: true
         opacity: state === "expanded" ? 0.6 : 0
         Behavior on opacity { NumberAnimation { duration: jobItem.animationDuration } }
 
         states: [
             State {
                 name: "expanded"
-                when: expandButton.checked && detailsLoader.status === Loader.Ready
+                when: detailsLoader.status === Loader.Ready
                 PropertyChanges {
                     target: detailsLoader
                     height: detailsLoader.item.implicitHeight
