--- /usr/share/plasma/plasmoids/org.kde.plasma.private.systemtray/contents/ui/ExpandedRepresentation.qml
+++ /usr/share/plasma/plasmoids/org.kde.plasma.private.systemtray/contents/ui/ExpandedRepresentation.qml
@@ -70,7 +70,6 @@ Item {
             }
         }
 
-        text: activeApplet ? activeApplet.title : i18n("Status & Notifications")
         MouseArea {
             anchors.fill: parent
             onClicked: {
@@ -116,7 +115,7 @@ Item {
         anchors {
             left: parent.left
             right: parent.right
-            top: heading.bottom
+            top: parent.top
             bottom: parent.bottom
             leftMargin: hiddenItemsView.visible ? hiddenItemsView.iconColumnWidth + units.largeSpacing : 0
         }
